autosklearn/automl.py

```diff
@@ -1476,19 +1496,15 @@ def fit_ensemble(
         precision: Literal[16, 32, 64] = 32,
         dataset_name: Optional[str] = None,
         ensemble_nbest: Optional[int] = None,
-        ensemble_size: Optional[int] = None,
+        ensemble_class: Optional[AbstractEnsemble] = EnsembleSelection,
+        ensemble_kwargs: Optional[Dict[str, Any]] = None,
     ):
```

Comment:
Hmmm, this makes it harder for a user to specify an ensemble size I guess, maybe `estimator.py` takes care of it but it would have to be well documented somewhere.

This also constitutes a breaking change. Maybe we can leave the parameter in `estimators.py` and just shove it into `ensemble_kwargs` with a warning about deprecation.

Reply:
Yes, deprecation is the right thing to do here. Thanks for the reminder. Will work on that.